elliott

```diff
@@ -131,12 +131,21 @@ Bugzilla Bugs.
 # advisory:create
 #
 @cli.command("create", short_help="Create a new advisory")
+@click.option("--type", '-t', 'errata_type',
+              type=click.Choice(['RHBA', 'RHSA', 'RHEA']),
+              default='RHBA',
+              help="Type of Advisory to create.")
 @click.option("--kind", '-k', required=True,
               type=click.Choice(['rpm', 'image']),
-              help="Kind of Advisory to create. Affects boilerplate text.")
+              help="Kind of artifacts that will be attached to Advisory. Affects boilerplate text.")
 @click.option("--impetus", default='standard',
               type=click.Choice(elliottlib.constants.errata_valid_impetus),
               help="Impetus for the advisory creation [standard, cve, ga, test]")
+@click.option('--impact', required=False,
+              type=click.Choice(['Critical', 'Important', 'Moderate', 'Low']),
+              help="Impact of the CVE. Required for RHSA.")
+@click.option("--cve", required=False,
```

Comment:
A Security Tracker blocks one or more parent BZ(s) (Flaw bugs) that each have a CVE alias. 
Each of the Flaw bugs should also be attached to the RHSA. 
Each CVE alias should be added to the RHSA's CVE Names field